specification/cognitiveservices/data-plane/Language/preview/2022-10-01-preview/analyzetext.json

```diff
@@ -0,0 +1,3039 @@
+        },
+        "parameters": {
+          "$ref": "#/definitions/LanguageDetectionTaskParameters"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeTextTask"
+        }
+      ],
+      "x-ms-discriminator-value": "LanguageDetection"
+    },
+    "AnalyzeTextSentimentAnalysisInput": {
+      "type": "object",
+      "properties": {
+        "analysisInput": {
+          "$ref": "#/definitions/MultiLanguageAnalysisInput"
+        },
+        "parameters": {
+          "$ref": "#/definitions/SentimentAnalysisTaskParameters"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeTextTask"
+        }
+      ],
+      "x-ms-discriminator-value": "SentimentAnalysis"
+    },
+    "AnalyzeTextDynamicClassificationInput": {
+      "type": "object",
+      "properties": {
+        "analysisInput": {
+          "$ref": "#/definitions/MultiLanguageAnalysisInput"
+        },
+        "parameters": {
+          "$ref": "#/definitions/DynamicClassificationTaskParameters"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeTextTask"
+        }
+      ],
+      "x-ms-discriminator-value": "DynamicClassification"
+    },
+    "AnalyzeTextJobsInput": {
+      "type": "object",
+      "properties": {
+        "displayName": {
+          "description": "Optional display name for the analysis job.",
+          "type": "string"
+        },
+        "defaultLanguage": {
+          "description": "Default language to use for records requesting automatic language detection.",
+          "type": "string"
+        },
+        "analysisInput": {
+          "$ref": "#/definitions/MultiLanguageAnalysisInput"
+        },
+        "tasks": {
+          "description": "The set of tasks to execute on the input documents.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AnalyzeTextLROTask"
+          }
+        }
+      },
+      "required": [
+        "analysisInput",
+        "tasks"
+      ]
+    },
+    "CustomTaskParameters": {
+      "type": "object",
+      "description": "Parameters object for a text analysis task using custom models.",
+      "properties": {
+        "projectName": {
+          "type": "string",
+          "description": "This field indicates the project name for the model."
+        },
+        "deploymentName": {
+          "type": "string",
+          "description": "This field indicates the deployment name for the model."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "common.json#/definitions/TaskParameters"
+        }
+      ],
+      "required": [
+        "projectName",
+        "deploymentName"
+      ]
+    },
+    "CustomResult": {
+      "type": "object",
+      "properties": {
+        "errors": {
+          "type": "array",
+          "description": "Errors by document id.",
+          "items": {
+            "$ref": "#/definitions/DocumentError"
```

Comment:
Question: I noticed that the definition of "PreBuiltResult" has a property called "errors" that changed from being an array of "DocumentErrors" to being "InputErrors". By any chance, was the intention to change that here too?

Reply:
I'm not the one made that change, but it looks like the refactoring happend in the same time new feature added with legacy 'DocumentErrors'. I agree we should consolidate the data structure to 'InputErrors' in the future.

Reply:
Yes, that will be a breaking SDK change.